Asociación de una propuesta con el Distrito del ciudadano

dany-herz42 opened the pull request:
### Cambios realizados
- Modificación de las Seed: Se modificaron las` seeds.rb`, esto debido a un error en el guardado del código del distrito en la BD
- Sobre-escritura de los siguientes métodos: 
```
update_proposal.rb
create_proposal.rb
publish_proposal.rb
```
En los anteriores 3 métodos se asocia la propuesta con el distrito del ciudadano que la creo.
ademas de la creacion de un metodo privado en cada una de las 3 clases para retornar el scope de `current_user`

dany-herz42 committed:
Actulizacion de las Seeds

- Las Seeds guardaban un hash, lo cual daba error. Ahora solo guardan un texto en codigo de distrito "code".

dany-herz42 committed:
Se sobrescribieron las clases "create_proposal.rb" y "publish_proposal.rb" para gregar el decidim_scope_id a la propuesta creada y/o publicada.

dany-herz42 committed:
Se sobrescribio la clase UpdateProposal para guardar el decidim_scope_id asociado a la propuesta actualizada.

oxcar commented:
No pasa el check de standardrb. 
Pueden volver a subirlo después de ejecutar:
`bundle exec standardrb --fix`

oxcar commented on lib/extensions/decidim/proposals/create_proposal.rb:
```diff
@@ -0,0 +1,47 @@
+              )
+              proposal.add_coauthor(@current_user, user_group: user_group)
+              proposal.scope = current_user_scope
+              binding.irb
```
Dejaron un binding.irb en el código

oxcar reviewed (changes requested):
Formatear el código con:
bundle exec standardrb --fix

Eliminar los binding.irb que se hayan quedado en el código

dany-herz42 committed:
Solucion de algunos errores:

- Ejecucion de bundle exec standardrb --fix
- Eliminacion de punto de interrupcion no necesario si no se esta depurando

dany-herz42 committed:
Se oculto el campo para que los usuarios seleccionaran su ambito.

oxcar reviewed (approved):
Todo bien
